extensions/wikia/Email/styles/main.css

```diff
@@ -1,6 +1,81 @@
-.content {
-	padding: 15px;
+/**
+ * This is the css file for the main.mustache template
+ * Note that it's a css file, not scss, so nesting and variables are not supported
+ * Only include styles that directly affect elements inside main.mustache
+ */
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.body-wrap {
+	width: 100%;
+}
+
+.main-wrapper {
+	background-color: #E6EBF2;
+}
+
+.main-content {
+	background-color: #FFF;
+	border-bottom: 1px solid #c5ced9;
+	border-top: 1px solid #c5ced9;
+	padding: 50px 40px 80px 40px;
+}
+
+.container {
+	display: block !important;
+	max-width: 600px !important;
+	margin: 0 auto !important;
+	clear: both !important;
+}
```

Comment:
Wow, are these '!important' statements essential here (even after inlining)? If so, maybe we could add a comment what do they need to override?